flixel/ui/FlxCircularBar.hx

```diff
@@ -0,0 +1,141 @@
+package flixel.ui;
+
+import flixel.FlxStrip;
+import flixel.graphics.FlxGraphic;
+import flixel.system.FlxAssets.FlxGraphicAsset;
+
+/**
+ * Circular progress bar.
+ * It is assumed that graphic of this object have equal width and height.
+ * @author Zaphod
+ */
+class FlxCircularBar extends FlxStrip 
```

Comment:
Does this really belong in core Flixel? There's already something very similar in flixel-addons (`flixel.addons.display.FlxPieDial`)...